resources/public/playground-dbg.html

```diff
@@ -7,16 +7,16 @@
 <link rel="stylesheet" type="text/css" href="css/codemirror.css">
 </head>
 <body>
-<pre><code class="javascript">
-    function foo (x) {
-    return x + 1;
+<pre><code class="ruby">
+    def foo (x) {
```

Comment:
It's not a valid ruby, Opal [can't parse it](http://opalrb.org/try/?code:def%20foo(x)%20%7B%0A%20%20x%20%2B%201%0A%7D%0A%20)